run_app.sh

```diff
@@ -1,22 +1,38 @@
-TEMPDIR=`mktemp -d`
+if [ -z "$ACG_CONFIG"]; then
```

Comment:
Does this come from clowder itself? Somehow automagically?

Reply:
@petrblaho Yes, this come from Clowder! This is inserted into the container so we can know if it's running with Clowder or not! Please look at:

https://clouddot.pages.redhat.com/clowder/dev/migration/migration.html#_code_changes_to_consume_configuration

Reply:
@jpramos123 thank you for your answer!